stdlib/public/core/Bitset.swift

```diff
@@ -179,8 +179,6 @@ extension _UnsafeBitset: Sequence {
   }
 }
 
-////////////////////////////////////////////////////////////////////////////////
```

Comment:
Seems like this ought to be a `// Mark -`

What do you think @lorentey?

Reply:
Where is there a precedent for using `// Mark -`?

Reply:
Apologies, `// MARK: -`

We use that quite frequently, but usually with some kind of title. On its own it will create a soft separator in Xcode.

Reply:
I do see it being used in some other places within the standard library, sometimes in conjunction with `===-------------===`. But do we really have an appropriate title?

If we do, it seems like refraining from `MARK` and just using plain `===---------===` is more consistent.

Reply:
I don't care about this -- any visual separator would work for me, but I wouldn't mind if this one got removed either. (If it was entirely up to me I'd've just used a FORM FEED character to insert a page break. Old habits die hard.)

@philipturner Does this comment actually interfere with anything? If not, it is very likely a better use of project resources to leave it as is. Arbitrary non functional changes are disruptive.

Reply:
It doesn't interfere with anything. I just saw it as wildly inconsistent with other documentation. Is there even another example of that in the Swift Standard Library?

Reply:
We don't have a strict coding guideline on how to section long source files -- whether to use such sectioning and what marks to use is (within reason!) left to the preferences and whims of individual contributors.

`// MARK:` comments have the minor benefit of triggering a mark in Xcode's source navigator pop up. (For developers who use Xcode to work on the stdlib.)

Purely stylistic changes have a tendency to generate large volumes of discussion, with very little actual benefit to the project. They can also confuse/obscure repository history with functionally irrelevant noise, and they often trigger unnecessary merge conflicts in downstream branches and repositories. Therefore, I find it's usually best not to merge such adjustments.

If you find the stdlib (or Swift in general) could use a strict coding style that leaves little room for such inconsistencies, I think the best way to achieve that is to work on swift-format. (I for one would very much love to have a rock solid formatter that we could use to automatically reformat/reflow the entire stdlib before every commit. Or, even better, to work on an AST-based source code representation that is free of any and all formatting choices -- indentation styles, line lengths, sectioning comments etc are presentation issues that would be best decided when the code is projected for display, based on the preferences of the user and the capabilities of the display device; they should not be baked into the actual source files. But I digress.)